library/systemd/test/systemd_service_test.rb

```diff
@@ -29,6 +29,23 @@ module Yast
       end
     end
 
+    describe ".build" do
```

Comment:
Actually this is an instance method.

Reply:
It is tricky: it looks like a `SystemdService` class method but it is a `SystemdServiceClass` instance method, as you say.